.github/workflows/tethys-release.yml

```diff
@@ -0,0 +1,98 @@
+        python-version: [3.7]
+    steps:
+      # Checkout the source
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      # Get Tag Name
+      - name: Get Tag
+        id: gitTag
+        uses: little-core-labs/get-git-tag@v3.0.2
+      # Build the docker for tag
+      - name: Build With Tag
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        run: |
+               docker build -t ${{ secrets.DOCKER_UPLOAD_URL }}:latest -t ${{ secrets.DOCKER_UPLOAD_URL }}:${{ steps.gitTag.outputs.tag }} .;
+      # Upload docker
+      - name: Upload Docker With Tag
+        if: ${{ github.event_name != 'pull_request' }} && ${{ startsWith(github.ref, 'refs/tags/') }} == "True"
+        run: |
+          echo "Pushing to docker registry";
+          echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin;
+          docker push ${{ secrets.DOCKER_UPLOAD_URL }}:${{ steps.gitTag.outputs.tag }};
+
+  tethys-conda:
+    name: Tethys Conda (${{ matrix.platform }})
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.7]
+    steps:
+      # Checkout the source
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      # Setup Tethys
+      - name: Setup Tethys
+        run: |
+          cd ..
+          bash ./tethys/scripts/install_tethys.sh --partial-tethys-install me -n tethys -s $PWD/tethys
+          . ~/miniconda/etc/profile.d/conda.sh
+          conda activate tethys
+          hash -r
+          conda config --set always_yes yes --set changeps1 no
+          conda update -q conda
+          conda install conda-build anaconda-client
+          conda config --set anaconda_upload no
+          mkdir -p ~/conda-bld
+      # Export Conda Build Path
+      - name: Set Conda Build Path
+        uses: allenevans/set-env@v2.0.0
+        with:
+          CONDA_BLD_PATH: '/home/runner/conda-bld'
+      # Generate Conda Recipe With Constrained Dependencies
+      - name: Generate Conda Recipe
+        run: |
+          cd ..
+          . ~/miniconda/etc/profile.d/conda.sh;
+          conda activate tethys;
+          tethys gen metayaml -p minor --overwrite;
+      # Show Tethys Meta
+      - name: Show Tethys Meta
+        run: |
+          cd ..
+          cat ./tethys/conda.recipe/meta.yaml
+      # Build Conda
+      - name: Build Conda
+        run: |
+          cd ..
+          . ~/miniconda/etc/profile.d/conda.sh;
+          conda activate tethys;
+          conda build -c tethysplatform -c conda-forge ./tethys/conda.recipe
+      # Upload Conda No Pull Request With Tag
+      - name: Upload Conda With Tag
+        run: |
+          cd ..
+          . ~/miniconda/etc/profile.d/conda.sh;
+          conda activate tethys;
+          anaconda -t "${{ secrets.CONDA_UPLOAD_TOKEN }}" upload -u aquaveo $CONDA_BLD_PATH/noarch/tethys-platform*.tar.bz2 --force;
```

Comment:
The user `aquaveo` is hard coded. Swap it out for an environment variable: CONDA_UPLOAD_USER

Reply:
done